hbase-compression/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <!--
+    /*
+     * Licensed to the Apache Software Foundation (ASF) under one
+     * or more contributor license agreements.  See the NOTICE file
+     * distributed with this work for additional information
+     * regarding copyright ownership.  The ASF licenses this file
+     * to you under the Apache License, Version 2.0 (the
+     * "License"); you may not use this file except in compliance
+     * with the License.  You may obtain a copy of the License at
+     *
+     *     http://www.apache.org/licenses/LICENSE-2.0
+     *
+     * Unless required by applicable law or agreed to in writing, software
+     * distributed under the License is distributed on an "AS IS" BASIS,
+     * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     * See the License for the specific language governing permissions and
+     * limitations under the License.
+     */
+  -->
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <artifactId>hbase-build-configuration</artifactId>
+    <groupId>org.apache.hbase</groupId>
+    <version>3.0.0-alpha-2-SNAPSHOT</version>
+    <relativePath>../hbase-build-configuration</relativePath>
+  </parent>
+  <artifactId>hbase-compression</artifactId>
+  <name>Apache HBase - Compression</name>
+  <description>Pure Java compression support parent</description>
+  <packaging>pom</packaging>
+  <modules>
+    <module>hbase-compression-aircompressor</module>
+    <module>hbase-compression-lz4</module>
+    <module>hbase-compression-snappy</module>
+    <module>hbase-compression-xz</module>
+    <module>hbase-compression-zstd</module>
+  </modules>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.hbase</groupId>
+      <artifactId>hbase-resource-bundle</artifactId>
+      <optional>true</optional>
+    </dependency>
+  </dependencies>
+
+  <properties>
+    <aircompressor.version>0.21</aircompressor.version>
+    <lz4.version>1.8.0</lz4.version>
+    <snappy.version>1.1.8.4</snappy.version>
+    <xz.version>1.9</xz.version>
+    <zstd-jni.version>1.5.0-4</zstd-jni.version>
+  </properties>
```

Comment:
Any chance they need to be shared across other modules? If so, good to keep them in parent pom.

Reply:
Sure, I will move them.